client/src/components/v3/Statistic.module.css

```diff
@@ -1,12 +1,20 @@
+.top {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+  align-items: flex-end;
+}
+
 .value {
   font-size: 32px;
-  line-height: 21px;
+  line-height: 34px;
   color: #000000;
+  text-align: right;
 }
 
 .unit {
   font-size: 14px;
-  line-height: 21px;
   color: #000000;
```

Comment:
`color: #000000;` is much harder to understand than `color: black`

Reply:
Good pick up! I have an extension that shows the colour for hex values so I've gotten used to working with hex as the default.
a2305a9